libminifi/include/core/state/nodes/AgentInformation.h

```diff
@@ -297,9 +297,20 @@ class ComponentManifest : public DeviceInformation {
             descriptorRequired.name = "required";
             descriptorRequired.value = prop.second.getRequired();
 
+            SerializedResponseNode descriptorDependentProperties;
+            descriptorDependentProperties.name = "dependentProperties";
+
+            for (const auto &propName : prop.second.getDependentProperties()) {
+              SerializedResponseNode descriptorDependentProperty;
+              descriptorDependentProperty.name = propName;
+              descriptorDependentProperty.value = propName;
```

Comment:
why is the value the property name? You don't need a value if you just need a list.

Reply:
Fixed.